shell/platform/windows/angle_surface_manager.cc

```diff
@@ -329,7 +329,7 @@ bool AngleSurfaceManager::MakeResourceCurrent() {
                          egl_resource_context_) == EGL_TRUE);
 }
 
-EGLBoolean AngleSurfaceManager::SwapBuffers() {
+bool AngleSurfaceManager::SwapBuffers() {
```

Comment:
What is the purpose of this change?

Reply:
Anytime we can avoid leaking implementation details (including types) from behind interfaces, it's a nice plus. Users of our surface managers shouldn't need to know about the underlying usage of GL types.